main.sh

```diff
@@ -167,7 +171,6 @@ main() {
   additionalParameters=$(echo "${workflowStepInputParams}" | jq -r '.additionalParameters')
 
   # Validate required parameters
-  [[ -z "${templateFile}" ]] && err "Template file is not passed in the Workflow Step inputs."
```

Comment:
Run a check that this is a file and not a dir and throw an error and contents of that folder so that user can see where in the tree are they.